### DISCUSSION TITLE
Need help deploying a project in a folder in a repo.

### DISCUSSION THREAD

**Discussion by omagarwal25** [vercel/community#978] [Help] [answered]
<img width="718" alt="Screenshot 2022-10-22 at 19 32 53" src="https://user-images.githubusercontent.com/25329723/197357159-82207978-6c23-4830-a92f-9983cfc4c1d5.png">

I'm trying to deploy my project. When is vercel doubling up on the /dash? This is what my config and repo look like.

<img width="853" alt="Screenshot 2022-10-22 at 19 33 33" src="https://user-images.githubusercontent.com/25329723/197357184-50696551-1215-4955-9bda-2521c625e47a.png">

<img width="922" alt="Screenshot 2022-10-22 at 19 33 55" src="https://user-images.githubusercontent.com/25329723/197357191-d23827d9-9588-4e07-bc8c-946bf34af297.png">

<img width="1270" alt="Screenshot 2022-10-22 at 19 34 07" src="https://user-images.githubusercontent.com/25329723/197357199-c3f9c6c6-3ccb-4cb2-be9a-4b0f5907853f.png">

It seems that Vercel is trying to find a tsconfig at /dash/dash. Why isn't it looking for it as /dash/tsconfig.json? Anyone have any ideas for fixes?

**Comment by amyegan** [comment]
Hi @omagarwal25! I'm guessing this is a monorepo, which might have it's own config interfering with your Project Settings. Can you share more about your project setup? I'm most interested in what monorepo tools you're using, your directory structure, and any related config.

**Comment by omagarwal25** [comment, marked as answer]
Hiya. I'm just going to copy-paste a write-up for anyone else with similar issues.

So it appears that NextJS runs eslint checks to ensure that everything is good before deploys. My Eslint looked something like this

{
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": "./dash/tsconfig.json"
  },
  "plugins": ["@typescript-eslint"],
  "extends": ["next/core-web-vitals", "plugin:@typescript-eslint/recommended"]
}


Notice the parserOptions. I had set it to that due to some TS issues I was encountering with the VSCode beta

Just fixing that line fixed it all

{
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": "./tsconfig.json" // this line right here
  },
  "plugins": ["@typescript-eslint"],
  "extends": ["next/core-web-vitals", "plugin:@typescript-eslint/recommended"]
}
This wasn't a TSC or Vercel issue at all. Just eslint being configured badly
Hope this helps someone in the future :)
